src/Response/FCC/AddRecords.php

```diff
@@ -21,6 +23,24 @@ public function __construct(array $recordsId)
         $this->recordsId = $recordsId;
     }
 
+    public static function fromArray(array $response): self
+    {
+        try {
+            Assert::keyExists($response, 'records_id');
+            Assert::isArray($response, 'records_id');
```

Comment:
Why you return `records_id` as the exception message? in `isArray` method. Looks like mistake.